IdentityCore/src/cache/token/MSIDCredentialCacheItem.m

```diff
@@ -285,12 +286,14 @@ - (BOOL)matchByEnvironment:(nullable NSString *)environment
     if (environment && 
         ![self.environment.msidNormalizedString isEqualToString:environment.msidNormalizedString])
     {
+        MSID_LOG_WITH_CTX(MSIDLogLevelVerbose, nil, @"(MSIDCredentialCacheItem) cached item did not have a valid environment. Actual: %@, expected: %@",self.environment, environment);
         return NO;
     }
 
     if ([environmentAliases count] && 
         ![self.environment.msidNormalizedString msidIsEquivalentWithAnyAlias:environmentAliases])
     {
+        MSID_LOG_WITH_CTX(MSIDLogLevelVerbose, nil, @"(MSIDCredentialCacheItem) cached item did not have a valid environment that matches with any other environment aliases.");
```

Comment:
Should we print aliases?

Reply:
I first thought it may not be needed, since the purpose was to identify the failing check, but sure there wouldn't be any harms in doing that.